assets/magnify.js

```diff
@@ -0,0 +1,42 @@
+// create a container and set the full-size image as its background
+function createOverlay(image) {
+  overlay = document.createElement('div');
+  overlay.setAttribute('class', 'image-magnify-full-size');
+  overlay.setAttribute('aria-hidden', 'true');
+  overlay.style.backgroundImage = `url('${image.src}')`;
```

Comment:
I tried this with an `img` element instead of a `div`, but the overlay image remains constrained in its container instead of being loaded at full size. Not sure what causes it or how to test whether it would load faster.

https://user-images.githubusercontent.com/22390758/197939841-0b16d25a-b2ce-474f-bf26-fd1a5ead2b9b.mp4